.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [15.x]
+        mongodb-version: ['6.0']
+    steps:
+      - name: Checking Repository
+        uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.8.0
+        with:
+          mongodb-version: ${{ matrix.mongodb-version }}
+          mongodb-port: 27017
+      - name: Cloning Repository
```

Comment:
Rename it to `Cloning Sugarizer Repository`. It's because `Checking Repository` is also cloning/checking out the repository.